### PR TITLE
[Fluid][RANS] Updated RANSApplication to work with Line and Surface conditions

### PR DESCRIPTION
**Pull request by sdharmin**
TESTS for FluidDynamicsApplication working.
TESTS for RANSApplication working.
Minor changes in code for the new Line and Surface conditions to work with RANSApplication

### CODE REVIEW

File: applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def __init__(self, model, custom_settings):
 
         ## Construct the turbulence model solver
         if not self.settings["turbulence_model_solver_settings"].IsEquivalentTo(KratosMultiphysics.Parameters("{}")):
-            self._turbulence_model_solver = CreateTurbulenceModel(model, self.settings["turbulence_model_solver_settings"])
+            self._turbulence_model_solver = CreateTurbulenceModel(model, self.main_model_part, self.settings["turbulence_model_solver_settings"])
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py]: Why do you need both the model and the model part? Shouldn't one of them be enough?

Comment (philbucher) [reply applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py]: same question

**Review by philbucher — CHANGES_REQUESTED** [review]
Either `Model` or `ModelPart` should be passed

**Comment by sunethwarna** [comment]
@philbucher @rubenzorrilla I changed it as you suggested. would you be able to look at it?

**Review by philbucher — APPROVED** [review]
:+1:

**Pull request merged**